sockeye/inference.py

```diff
@@ -1619,7 +1603,7 @@ def _beam_search(self,
         scores_accumulated = scores_accumulated.take(best_hyp_indices)
         constraints = [constraints[x] for x in best_hyp_indices.asnumpy()]
 
-        return sequences, attentions, scores_accumulated, lengths, constraints, beam_histories
+        return sequences[:,1:], attentions[:,1:,:], scores_accumulated, lengths, constraints, beam_histories
```

Comment:
maybe add a comment that we're removing the sentence start symbol here

Reply:
Fixed. We could also just not add these columns when we initialize sequences and attentions, but it seems you can't create and then later extend an `ndarray` with a 0 dimension.